### PR TITLE
presumptive rr not on encounters#show

### PR DESCRIPTION
**Pull request by jackregnart**
Missed a file. Display Presumptive RR on `encounters#show`.

### CODE REVIEW

File: cdx_core/app/assets/javascripts/components/encounter_show.js.jsx [hunk 1]
```diff
@@ -98,7 +98,7 @@ var EncounterShow = React.createClass({
                 }
                 {
                   this.props.encounter.presumptive_rr ?
-                  <DisplayFieldWithLabel fieldLabel={ I18n.t("components.encounter_show.presumptive") } fieldValue={ this.props.encounter.presumptive_rr } />
+                  <DisplayFieldWithLabel fieldLabel={ I18n.t("components.encounter_show.presumptive") } fieldValue={ this.props.encounter.presumptive_rr ? 'Yes' : 'No' } />
```
Comment (jorgegorka) [inline cdx_core/app/assets/javascripts/components/encounter_show.js.jsx]: You need to translate Yes and No

File: cdx_vietnam/app/assets/javascripts/components/encounter_show.js.jsx [hunk 2]
```diff
@@ -124,7 +124,7 @@ var EncounterShow = React.createClass({
                 }
                 {
                   this.props.encounter.presumptive_rr ?
-                  <DisplayFieldWithLabel fieldLabel={I18n.t("components.encounter_show.presumptive")} fieldValue={ this.props.encounter.presumptive_rr } />
+                  <DisplayFieldWithLabel fieldLabel={ I18n.t("components.encounter_show.presumptive") } fieldValue={ this.props.encounter.presumptive_rr ? 'Yes' : 'No' } />
```
Comment (jorgegorka) [inline cdx_vietnam/app/assets/javascripts/components/encounter_show.js.jsx]: Same here

File: cdx_core/app/assets/javascripts/components/encounter_show.js.jsx (hunk 1 above)
Comment (jackregnart) [inline cdx_core/app/assets/javascripts/components/encounter_show.js.jsx]: Ah yes. 👍

Comment (jackregnart) [inline cdx_core/app/assets/javascripts/components/encounter_show.js.jsx]: Amended

**Pull request merged**